### PR TITLE
feat: add GuardDuty Findings support

### LINKED ISSUE #48: Lambda error with GuardDuty findings
Hi all,

I have set up this module to get slack notifications when a GuardDuty finding is found. There is a CW event that publishes the message to the SNS topic. I generate the samples findings from the GuardDuty console but nothing arrives. Looking to the lambda execution logs, I found this errors:

```
15:43:05
START RequestId: 10f8c40c-7e40-485f-a04e-1a9ae36c3586 Version: $LATEST

15:43:05
[ERROR] 2019-11-20T15:43:05.132Z 10f8c40c-7e40-485f-a04e-1a9ae36c3586 JSON decode error: Expecting value: line 1 column 1 (char 0)

15:43:05
Traceback (most recent call last):

15:43:05
File "/var/task/notify_slack.py", line 64, in notify_slack

15:43:05
message = json.loads(message)

15:43:05
File "/var/lang/lib/python3.7/json/__init__.py", line 348, in loads

15:43:05
return _default_decoder.decode(s)

15:43:05
File "/var/lang/lib/python3.7/json/decoder.py", line 337, in decode

15:43:05
obj, end = self.raw_decode(s, idx=_w(s, 0).end())

15:43:05
File "/var/lang/lib/python3.7/json/decoder.py", line 355, in raw_decode

15:43:05
raise JSONDecodeError("Expecting value", s, err.value) from None

15:43:05
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

15:43:05
[ERROR] HTTPError: HTTP Error 404: Not Found Traceback (most recent call last):   File "/var/task/notify_slack.py", line 84, in lambda_handler     notify_slack(subject, message, region)   File "/var/task/notify_slack.py", line 77, in notify_slack     urllib.request.urlopen(req, data)   File "/var/lang/lib/python3.7/urllib/request.py", line 222, in urlopen     return opener.open(url, data, timeout)

15:43:05
END RequestId: 10f8c40c-7e40-485f-a04e-1a9ae36c3586

15:43:05
REPORT RequestId: 10f8c40c-7e40-485f-a04e-1a9ae36c3586 Duration: 381.32 ms Billed Duration: 400 ms Memory Size: 128 MB Max Memory Used: 72 MB Init Duration: 329.24 ms
```

I supposed this lambda is not made for parsing GD findings. 
Any workaround?

### PR DESCRIPTION
**Pull request by iogi**
## Description
Support GuardDuty finding events
https://docs.aws.amazon.com/guardduty/latest/ug/guardduty_findings_cloudwatch.html

## Motivation and Context
This PR closes #48

## Breaking Changes
This PR improves GuardDuty findings event outputs and affects only GuardDuty finding events.

## How Has This Been Tested?

Run modified function using AWS-provided sample findings.
https://docs.aws.amazon.com/guardduty/latest/ug/sample_findings.html

image

### CODE REVIEW

**Comment by codigube** [comment]
when can we merge this?

File: functions/notify_slack.py [hunk 1]
```diff
@@ -41,6 +41,45 @@ def cloudwatch_notification(message, region):
   }
 
 
+def guardduty_finding(message):
+  states = {'Low': '#777777', 'Medium': 'warning', 'High': 'danger'}
+  region = message['region']
```
Comment (pjaudiomv) [inline functions/notify_slack.py:46]: any reason you didn't pass the region into the function similar to the cloudwatch_notification one

File: functions/notify_slack.py [hunk 2]
```diff
@@ -41,6 +41,45 @@ def cloudwatch_notification(message, region):
...
+
+  if region.startswith("us-gov-"):
+    cloudwatch_url = "https://console.amazonaws-us-gov.com/guardduty/home?region="
```
Comment (pjaudiomv) [inline functions/notify_slack.py:49]: these could be better named to what they are, guardduty_url

**Comment by pjaudiomv** [comment]
I would add a test fixture as well

**Comment by antonbabenko** [comment]
Fixed in #143.

[v4.18.0](https://github.com/terraform-aws-modules/terraform-aws-notify-slack/releases/v4.18.0) has been just released.

**Pull request closed without merging**